src/types/table.ts

```diff
@@ -17,6 +17,9 @@ export interface TablePreferences {
 
 export type TableViewData = Record<string, any>[]
 
+/**
+ * @deprecated in favor of TableDataHeader
+ */
```

Comment:
issue: it doesn't make sense to deprecate this interface if the `TableDataHeader` is still extending it

Reply:
Once we are ready to remove it we can just swap it without changing any of the properties

Reply:
You can't extend a deprecated interface in an interface you are _not_ deprecating. 

If you're wanting to rename, you'll need to copy/paste the interface into the new one, and mark the old one as deprecated.

Reply:
Done